README.md

```diff
@@ -93,6 +141,7 @@ The adaptor configuration has sensible defaults for local development. Some over
 
```

Comment:
Can we add the following lines after `Verify the project structure` to provide an example of the structure?

```suggestion
    Project structure -> SDKs -> add new SDK -> select adoptopenjdk-11.jdk/Contents/Home  (or alternative location)
                      -> Project SDK -> Java 11 (11.0.7)
                      -> Module SDK -> Java 11 (11.0.7)

```


![image](https://user-images.githubusercontent.com/74314114/107974915-31c6bc80-6faf-11eb-92cc-7cd0d9e296f1.png)

Reply:
Yes, of course 👍